### ISSUE TITLE
[3.9.13] [AspNetCore] .ts file generated by sergen not complete

### ISSUE THREAD

**Issue by Davidwsk** [volkanceylan/Serenity#4780]
Hi,

I am a new user.
Trying out the Tutorial.
After generate the initial code using "dotnet sergen g"
Both .ts files are not complete.
Missing the closing "}}"
![Sergen TS](https://user-images.githubusercontent.com/31944359/71051516-8c823f80-2183-11ea-90d7-73a73d137b40.png)

But it can still work somehow...
Is it normal?
What should the files be?

Thanks!

**Comment by issamogbah** [comment]
Hello,

The same with ASP.Net 

All Dialog and Grid generated typescript files are incomplete..

Any fix?

**Comment by Davidwsk** [comment]
After some checking.
I found out that these 2 template files have error:
https://github.com/volkanceylan/Serenity/blob/master/Serenity.CodeGenerator/Templates/Dialog.scriban
https://github.com/volkanceylan/Serenity/blob/master/Serenity.CodeGenerator/Templates/Grid.scriban
There are extra {{end}} which course error of TS.

And there is a typo-error in Dialog.scriban, "getDeletePermission" should return "deletePermission" but not "insertPermission".

```
protected getDeletePermission() { return {{RowClassName}}.insertPermission; }{{end}}
protected getInsertPermission() { return {{RowClassName}}.deletePermission; }{{end}}
protected getUpdatePermission() { return {{RowClassName}}.updatePermission; }{{end}}

```

My work around is manually modify the TS file base on these 2 template files.

**Comment by volkanceylan** [comment]
Oh, my bad sorry, will fix them soon.

**Comment by volkanceylan** [comment]
Ok this should be fixed in 3.9.14, please test and let me know if any issues
